config/main.py

```diff
@@ -1404,5 +1404,54 @@ def del_syslog_server(ctx, syslog_ip_address):
     except SystemExit as e:
         ctx.fail("Restart service rsyslog-config failed with error {}".format(e))
 
+#
+# 'ntp' group ('config ntp ...')
+#
+@config.group()
+@click.pass_context
+def ntp(ctx):
+    """NTP server configuration tasks"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+    pass
+
+@ntp.command('add')
+@click.argument('ntp_ip_address', metavar='<ntp_ip_address>', required=True)
+@click.pass_context
+def add_ntp_server(ctx, ntp_ip_address):
+    """ Add NTP server IP """
+    if not is_ipaddress(ntp_ip_address):
+        ctx.fail('Invalid ip address')
+    db = ctx.obj['db']
+    ntp_servers = db.get_table("NTP_SERVER")
+    if ntp_ip_address in ntp_servers:
+        click.echo("NTP server {} is already configured".format(ntp_ip_address))
+        return
+    else: 
+        db.set_entry('NTP_SERVER', ntp_ip_address, {'NULL': 'NULL'})
+        click.echo("NTP server {} added to configuration".format(ntp_ip_address))
+        try:
+            click.echo("Restarting ntp-config service...")
+            run_command("systemctl restart ntp-config", display_cmd=False)
+        except SystemExit as e:
+            ctx.fail("Restart service ntp-config failed with error {}".format(e))
+
+@ntp.command('del')
+@click.argument('ntp_ip_address', metavar='<ntp_ip_address>', required=True)
+@click.pass_context
+def del_ntp_server(ctx, ntp_ip_address):
+    """ Delete NTP server IP """
+    if not is_ipaddress(ntp_ip_address):
+        ctx.fail('Invalid IP address')
+    db = ctx.obj['db']
+    db.set_entry('NTP_SERVER', '{}'.format(ntp_ip_address), None)
```

Comment:
I think we should check here (like we did for the DHCP relay destination IPs: https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L886) to see if the address we are trying to delete to check if it exists and report to the user if we can't delete it because it doesn't exist.

I think you should also open a separate PR to do the same for the syslog servers.

Reply:
updated.